### ISSUE TITLE
How to train both face detection (regression) and facial expression recognition (classification) using DarkFlow?

### ISSUE THREAD

**Issue by aaronhan92** [thtrieu/darkflow#913]

**Comment by aaronhan92** [comment]
Please kindly comment whether if my implementation flow works as follow:

1) Git clone the darkflow.git

2) Download pre-trained tiny-yolo-widerface.cfg and tiny-yolo-widerface_final.weights from https://drive.google.com/file/d/1wAy6-XWDBHMMawgPfKv2hp_Q5NBYqK1n/view (Able to detect face with bounding box that trained on WiderFace dataset)

3) Prepare my own small dataset (also copy from Widerface) using LabelImg software tool to crop and label each facial expression such as happy and sad etc.

4) Start to train my model using this command 'python flow --model cfg/tiny-yolo-widerface.cfg --train --trainer adam --annotation train_set/Annotation --dataset training_set/Images'.

**Comment by sandeeprepakula** [comment]
Do you have the labeled data for training? I mean the ground truth top left and bottom right co-ordinates of the faces and their corresponding expression labels like smiling, angry, etc?
